app/controllers/miq_policy_controller/alert_profiles.rb

```diff
@@ -54,6 +54,7 @@ def alert_profile_edit
                                                 _("%{model} \"%{name}\" was added")
         add_flash(flash_key % {:model => ui_lookup(:model => "MiqAlertSet"), :name => @edit[:new][:description]})
         alert_profile_get_info(MiqAlertSet.find(alert_profile.id))
+        alert_profile_sync_provider(current, mems.keys)
         @edit = nil
         self.x_node = @new_alert_profile_node = "xx-#{alert_profile.mode}_ap-#{to_cid(alert_profile.id)}"
```

Comment:
It is part of the UI symbols/labels where tree elements are identified in the controller part.
The Alert Profile UI has a tree component to select which elements applied to the Alert Profile.